frontend/src/shared/helpers.tsx

```diff
@@ -889,3 +947,7 @@ export function handleSearch(
   const parsedSearchVals = parseUserSearchVal(userSearchVal);
   setParsedSearchVals(parsedSearchVals);
 }
+
+function formatCohortRelatedDate(date: string) {
+  return date?.split(" ")[0];
+}
```

Comment:
These changes would remove the hour and minute from the timestamps. This is a simple change so I figured I could squeeze them into this PR.